### PR TITLE
show voice indicator above dominant jitsi speaker

### PR DESCRIPTION
**Pull request by Waclaw-I**

### CODE REVIEW

**Comment by moufmouf** [comment]
One issue I see: if the user leaves the room while being dominant, the speak indicator stays displayed.

**Comment by moufmouf** [comment]
One last small change:

could you add a check so that if the user is left alone in the room, the speaker indicator is removed?

It's a bit weird to see the speak indicator staying up indefinitely and even especially when there is only one person left in the room :)

File: front/src/Phaser/Game/GameScene.ts [hunk 1]
```diff
@@ -673,6 +668,18 @@ export class GameScene extends DirtyScene {
...
         });
 
+        this.userIsJitsiDominantSpeakerStoreUnsubscriber = userIsJitsiDominantSpeakerStore.subscribe(
+            (dominantSpeaker) => {
+                this.jitsiDominantSpeaker = dominantSpeaker;
+                this.tryChangeShowVoiceIndicatorState(this.jitsiDominantSpeaker && this.jitsiParticipantsCount > 1);
+            }
+        );
+
+        this.jitsiParticipantsCountStoreUnsubscriber = jitsiParticipantsCountStore.subscribe((participantsCount) => {
```
Comment (moufmouf) [inline front/src/Phaser/Game/GameScene.ts:678]: The unsubscriber is missing ;)

Comment (Waclaw-I) [reply front/src/Phaser/Game/GameScene.ts:678]: AaaH!

**Pull request merged**